### ISSUE TITLE
How to config nuxt3 to generate static site so that route page is loaded by partially refreshing and without useFetch request?

### ISSUE THREAD

**Issue by tjswk2008** [nuxt/nuxt#29914]
### Environment

------------------------------
- Operating System: Windows_NT
- Node Version:     v16.20.1
- Nuxt Version:     3.7.4
- CLI Version:      3.14.0
- Nitro Version:    2.9.7
- Package Manager:  pnpm@6.15.0
- Builder:          -
- User Config:      -
- Runtime Modules:  -
- Build Modules:    -
------------------------------

### Reproduction

In nuxt3, inner route page still process useFetch ajax request after generation.

### Describe the bug

How to config nuxt3 to generate static site so that route page is loaded by partially refreshing and without useFetch request?
In nuxt2, I can set target: 'static' to implement this feature.

### Logs

```shell-script

```

**Comment by Norbiros** [comment]
@fhqiypmb could you please elaborate on this issue? The fetch content is extracted (it is wrapped with `useAsyncData`) and reused on client, so the fetch is never duplicated on static/generated sites.

**Comment by danielroe** [comment]
yes, I think this is implemented already if you prerender your site, or part of your site.

for part of your site, just set:

```ts
export default defineNuxtConfig({
  nitro: {
    prerender: {
      routes: ['/route-you-want-to-prerender']
    }
  }
})
```

for all of your site, you can just run `nuxt generate`

nuxtathon closed @norbiros
